VelocidiSDK/VelocidiSDK/VSDKRequest.h

```diff
@@ -6,6 +6,9 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+  Class that wraps a NSURLMutableRequest with the necessary addons to make a request to Velocidi's infrastructure
```

Comment:
Seems more accurate to me. What do you think?

(This suggestion also applies to other places)

```suggestion
  Class that wraps a NSURLMutableRequest with the necessary addons to make a request to Velocidi's system
```